pkg/slo-controller/noderesource/plugins/cpunormalization/plugin.go

```diff
@@ -174,10 +174,11 @@ func (p *Plugin) Calculate(_ *configuration.ColocationStrategy, node *corev1.Nod
 		return nil, fmt.Errorf("failed to get CPUBasicInfo in cpu normalization calculation, err: info is missing")
 	}
 
-	ratio, err := getCPUNormalizationRatioFromModel(basicInfo, strategy)
+	ratio, err := getCPUNormalizationRatio(basicInfo, strategy)
 	if err != nil {
-		return nil, fmt.Errorf("failed to get ratio in cpu normalization calculation, err: %s", err)
+		return nil, fmt.Errorf("failed to get cpu normalization ratio for node %s, err: %w", node.Name, err)
```

Comment:
No need to wrap the nodeName in the error again, since the framework can print it.